otherlibs/dune-rpc/private/dune_rpc_private.mli

```diff
@@ -406,6 +406,11 @@ module Public : sig
     val ping : (unit, unit) t
 
     val diagnostics : (unit, Diagnostic.t list) t
+
+    val format_dune_file :
+      ([ `Path of string ] * [ `Contents of string ], string) t
```

Comment:
It feels a bit inconsistent to give a regular pair of polymorphic variants here, but using regular variant types elsewhere.

Reply:
I can introduce a dedicated record for this if that's what you prefer.

Reply:
After more thought, I don't think it's a big deal. It's less cluttered to do it this way, so may as well leave it.

Reply:
Actually, I like introducing a record here. I expect that it will be used in other calls.